app/search/flows/next-page-viewer-search.ts

```diff
@@ -46,7 +46,7 @@ function nextPageArgs(
           .time(prevTs)
           .add(1, "ms")
           .toDate()
-        spliceIndex = index + 1
+        spliceIndex = index
```

Comment:
I think this is right. I saw a bug where the last record was duplicated. Need to confirm.